book.common/src/main/java/com/happystays/book/common/dto/eventstoremodel/PnrEventModel.java

```diff
@@ -0,0 +1,21 @@
+package com.happystays.book.common.dto.eventstoremodel;
```

Comment:
no need to keep this in dto I think
1. Rename as PnrEvent
2. we have a package events in common keep it there